system/core/dynamic/HTMLHelper.cfc

```diff
@@ -1131,8 +1131,10 @@ Description :
 					if( len( arguments.nameColumn ) ){
 						if( structKeyExists( val[x], arguments.nameColumn ) ){ thisName = val[x][nameColumn]; }
 					}
-					else{
-						if( structKeyExists( val[x], arguments.column ) ){ thisName = val[x][column]; }
+					
+					// Only set to the value if thisName is still empty
```

Comment:
What about the if statement that checks if the value has a `column` field? You ignore it

Reply:
No, that still does it’s thing.

Reply:
Where? The code is removed